deploy_and_run_tests.sh

```diff
@@ -226,22 +224,10 @@ if [[ -z "${skip_build}" ]]; then
       --substitutions _CUSTOM_TAG_NAME="${image_tag}" .
 fi
 
-# Running integration tests in a temporary virtualenv
```

Comment:
This was actually a very useful capability to be able to run tests on our machines without 100% relying cloud build.
Do you think there is any value to comment out this section instead of deleting it? I am hopeful in future we might spend time on this and fix it so we can run `./deploy_and_run_tests.sh` from our workstations. What do you think?